python_docs_theme/static/pydoctheme.css

```diff
@@ -140,6 +140,8 @@ span.pre {
 }
 
 div.sphinxsidebar {
+    display: flex;
```

Comment:
Is flexbox required here?

Reply:
This allows me to remove the hardcoded `width: 217px;` from `div.sphinxsidebarwrapper`